src/appmixer/freshdesk/tickets/DeletedTicket/DeletedTicket.js

```
@@ -0,0 +1,62 @@
+                    id: ticket.id,
+                    subject: ticket.subject,
+                    deleted_at: ticket.updated_at
+                });
+            }
+        });
+
+        if (deletedTickets.length) {
+            deletedTickets.forEach(ticket => {
+                context.sendJson(ticket, 'ticket');
+            });
+        }        
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)